.pnp.loader.mjs

```diff
@@ -0,0 +1,2047 @@
+import { URL as URL$1, fileURLToPath, pathToFileURL } from 'url';
```

Comment:
What is this pnp (and other related files) for?

Also do you think they should be pushed, or be here at all?